pkg/e2e/e2e.go

```diff
@@ -105,7 +105,7 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 		viper.Set(config.CloudProvider.Region, cluster.Region())
 		log.Printf("CLOUD_PROVIDER_REGION set to %s from OCM.", viper.GetString(config.CloudProvider.Region))
 
-		if !viper.GetBool(config.Addons.SkipAddonList) {
+		if !viper.GetBool(config.Addons.SkipAddonList) || viper.GetString(config.Provider) == "mock" {
```

Comment:
Why do this when the provider *is* the mock provider? It seems to me that you'd want the opposite

Reply:
You are _so_ so right.

Reply:
Fixed! PTAL